src/adapters/polkadot.ts

```diff
@@ -181,7 +190,8 @@ class BasePolkadotAdapter extends BaseCrossChainAdapter {
     const { address, amount, to, token } = params;
     const toChain = chains[to];
 
-    if (token !== this.balanceAdapter?.nativeToken) {
+    // fixme: should check `token !== this.balanceAdapter?.nativeToken` instead
```

Comment:
We currently can't do this fix because a small problem with our testnet Kusama deploy: the native token's symbol is "UNIT" so this comparison fails